crates/utils/src/tracing_util.rs

```diff
@@ -1,19 +1,39 @@
 //! Helpers related to tracing, used by main entrypoints
 
+use tracing_subscriber::prelude::*;
+
 /// Initialize tracing with the default configuration.
 pub fn initialize_tracing() {
-    // Don't include timestamps and such because they're not really useful and
-    // too verbose, and plus several log targets such as journald will already
-    // include timestamps.
+    // Always try to use journald subscriber if we're running under systemd
+    // This ensures key messages (info, warn, error) go to the journal
+    let journald_layer = if let Ok(()) = std::env::var("JOURNAL_STREAM").map(|_| ()) {
+        tracing_journald::layer().ok()
```

Comment:
I think we want to have a filter here so that only info and above goes to the journal, right? I think it's possible with https://docs.rs/tracing-subscriber/latest/tracing_subscriber/layer/trait.Layer.html#method.with_filter - I can take a look if you want.

Otherwise someone debugging with e.g. `env RUST_LOG=trace` will completely spam the journal.